vignettes/.gitignore

```diff
@@ -1,2 +1,3 @@
 *.html
 *.R
+^vignettes/meta-forestly.Rmd$
```

Comment:
This should be added to `.Rbuildignore`, not `.gitignore`.

Reply:
done